mne/viz/ica.py

```diff
@@ -627,30 +633,48 @@ def plot_ica_scores(ica, scores, exclude=None, labels=None, axhline=None,
     exclude = np.unique(exclude)
     if not isinstance(scores[0], (list, np.ndarray)):
         scores = [scores]
-    n_rows = len(scores)
+    n_scores = len(scores)
+
+    if n_cols is None:
+        # prefer more rows.
+        n_rows = int(np.ceil(np.sqrt(n_scores)))
+        n_cols = int(np.ceil(n_scores / float(n_rows)))
+    else:
+        n_cols = min(n_scores, n_cols)
+        n_rows = int(np.ceil(n_scores / float(n_cols)))
```

Comment:
It's better to stick with int arithmetic where possible, so:
```
n_rows = (n_scores - 1) // n_cols + 1
```
same in the `n_cols` calculation above

Reply:
... but it would be nice to move to int arithmetic here still